tests/snapshots/tests.contrib.wsgi.test_wsgi.test_generator_exit_ignored_in_top_level_span_snapshot_py2.json

```diff
@@ -2,15 +2,17 @@
   {
     "name": "wsgi.request",
     "service": "wsgi",
-    "resource": "wsgi.request",
+    "resource": "GET /generatorError",
```

Comment:
Bug fix: we should always set the correct resource even when an exception is raised by the endpoint. I would open a seperate PR for this fix but I need to test this fix with flask (and I already have 2 PRs open for this escalation)